nomad/structs/structs.go

```diff
@@ -2225,6 +2230,7 @@ func (n *Node) Stub(fields *NodeStubFields) *NodeListStub {
 type NodeListStub struct {
 	Address               string
 	ID                    string
+	Attributes            map[string]string
```

Comment:
After adding the `if` check from the comment above add the `json:",omitempty"` tag here to skip serializing this field on empty responses.

Reply:
Fixed.